src/main/java/edu/indiana/dlib/amppd/model/Passwordresettoken.java

```diff
@@ -33,7 +33,7 @@
     private String token;
   
     @OneToOne(targetEntity = AmpUser.class, fetch = FetchType.EAGER)
-    @JoinColumn(nullable = false, name = "id")
+    @JoinColumn(nullable = false, name = "user_id")
     private AmpUser user;
```

Comment:
Just to double check that this change is intentional? I remembered we had some issue with user_id. is the column name changed to user_id?

Reply:
yes